src/modules/ui/components/CardDetail/CardDetail.jsx

```diff
@@ -8,6 +8,7 @@ import { FormattedMessage, FormattedDate } from 'react-intl';
 import * as felaRules from './CardDetail.styles';
 import Button from '../Button';
 import NoteForm from '../NoteForm';
+import DeleteButton from '../../../reports/components/DeleteButton';
```

Comment:
This is imported from different module so it should be like this
```suggestion
import { DeleteButton } from 'modules/reports';
```

Reply:
be sure DeleteButton is exported from that module via its index.js